app/controllers/sign_in_tokens_controller.rb

```diff
@@ -19,16 +19,30 @@ def manual
 
   def create
     @sign_in_token_form = SignInTokenForm.new(sign_in_token_form_params)
+    redirect_to new_user_path and return if @sign_in_token_form.already_have_account == 'no'
 
-    if @sign_in_token_form.valid? && @sign_in_token_form.email_is_user?
-      token = SessionService.send_magic_link_email!(@sign_in_token_form.email_address)
-      # NOTE: this is purely to allow us to display the link in the footer
-      # REMOVE when we have emails actually being sent
-      identifier = User.where(sign_in_token: token).first.try(:sign_in_identifier, token)
+    if @sign_in_token_form.valid?
+      token = if @sign_in_token_form.email_is_user?
```

Comment:
I found the `email_is_user?` method name cryptic and had to look it up in the code to figure out what it did.